CHANGELOG.unreleased.md

```diff
@@ -7,6 +7,8 @@
 
 > Users must be able to say: “Nice enhancement, I'm eager to test it”
 
+- [Backup] Go back to previous page instead of going to the overview when saving or creating a backup job: keeps the user's current filters and page (PR [#5913](https://github.com/vatesfr/xen-orchestra/pull/5913))
```

Comment:
Job creation is no longer impacted:

```suggestion
- [Backup] Go back to previous page instead of going to the overview after editing a job: keeps current filters and page (PR [#5913](https://github.com/vatesfr/xen-orchestra/pull/5913))
```

Reply:
Not fixed.